pantheon/src/main/java/tech/pegasys/pantheon/controller/CliquePantheonController.java

```diff
@@ -142,7 +142,8 @@
             protocolContext.getBlockchain(),
             networkId,
             fastSyncEnabled,
-            syncConfig.downloaderParallelism());
+            syncConfig.downloaderParallelism(),
+            syncConfig.transactionsParallelism());
```

Comment:
Nitpick (optional) - it looks like SyncConfig is becoming EthConfig - maybe we should rename and move it??

Reply:
There may be more to it than a surface renaming.  Spun off to NC-1993.